### PR TITLE
Added an option to hide album artwork

### PR DESCRIPTION
**Pull request by avgurf**
- resolved issue #11  
- Added a "Hide artwork" option in landing.js
- Added a "settings" link in the bottom of app.js to return back to change settings
- Changed routing to keep user settings when clicking on settings link, and to rout user back to landing.js in case of bad prams and 404s

image

image

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -18,10 +18,12 @@ body {
...
- }  
+ }
 }
 
 .github-icon {
   width: 20px;
   height: 20px;
 }
+
+.hidden { display: none !important;}
```
Comment (guytepper) [inline src/index.css]: Don't use `!important`

File: src/App.js [hunk 2]
```diff
@@ -170,7 +171,7 @@ class App extends Component {
       return (
         <div>
           <h1>{ this.state.error }.</h1>
-          <Link to={'/'}>
+          <Link to={`/${this.username}/${this.period}/${this.state.isHardMode}`}>
```
Comment (guytepper) [inline src/App.js]: Doesn't work correctly - `isHardMode` is passed wrong

File: src/App.js [hunk 3]
```diff
@@ -192,6 +193,7 @@ class App extends Component {
         { this.gameEndMessage() }
         { this.playAgainBtn() }
         <Keyboard onPress={ this.handleLetterGuess } />
+        <Link className='game-change-settings-link' to={`/${this.username}/${this.period}/${this.state.isHardMode}`}>Settings</Link>
```
Comment (guytepper) [inline src/App.js]: Can you add some styles to it?

File: src/components/Artwork.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-function Artwork ({ img, blurLevel, gameEnd }) {
+function Artwork ({ img, blurLevel, gameEnd, hidden }) {
```
Comment (guytepper) [inline src/components/Artwork.js]: `hidden` -> `isHidden`

File: src/components/Artwork.js [hunk 5]
```diff
@@ -20,7 +20,7 @@ function Artwork ({ img, blurLevel, gameEnd }) {
  };
 
   return (
-     <img style={ artworkStyle } src={ img } alt=""/>
+     <img style={artworkStyle} className={hidden? 'hidden' : ''} src={ img } alt=""/>
```
Comment (guytepper) [inline src/components/Artwork.js]: Maybe let's just use inline styles here?

File: src/components/Artwork.js (hunk 4 above)
Comment (avgurf) [reply src/components/Artwork.js]: I think we better stick to `hidden `to align with the HTML attribute

File: src/components/Landing.js [hunk 6]
```diff
@@ -61,12 +72,24 @@ class Landing extends React.Component {
           </label>
         </div>
 
-        <Link to={`/game/${this.state.username}/${this.state.period}`} style={!this.state.username ? {pointerEvents: "none"} : null}>
-          <button className='button-success pure-button' disabled={!this.state.username}>Let's play! <span role="img" aria-label="Clown">🤡</span></button>
+        <div className='langing-hard-mode'>
+          <lable>
+            <input type="checkbox" onChange={this.handleHardModeChange}
+              name="hardmode"  defaultChecked={this.state.isHardMode} />
```
[the remark this answers is not in the document]
Comment (avgurf) [reply src/components/Landing.js]: When using checked then the checked state is affected according to the state itself, which wouldn't allow unchecking the checkbox and triggering the `onChange` event

**Comment by avgurf** [comment]
This had become too messy (and also turned out to be a total mistake to pull it on the master branch of my fork repo 💩), I'll create a new one instead 😎

**Pull request closed without merging**